### PR TITLE
Complete KeyBindingsGuiTest

### PR DESCRIPTION
**Pull request by damithc**
Added some missing tests

### CODE REVIEW

File: src/test/java/guitests/guihandles/MainGuiHandle.java [hunk 1]
```diff
@@ -23,4 +23,18 @@ public MainMenuHandle getMainMenu() {
...
+
+    public boolean isMaximized() {
+        return true;
+        //TODO: implement this
+    }
+
+    public boolean isDefaultSize() {
+        return true;
+        //TODO: implement this
+    }
```
Comment (damithc) [inline src/test/java/guitests/guihandles/MainGuiHandle.java]: @m133225 I need your help to complete these three methods.

File: src/test/java/address/util/TestUtil.java [hunk 2]
```diff
@@ -240,4 +240,15 @@ public static void initBrowserInStatic() {
                 1, BrowserType.FULL_FEATURE_BROWSER);
         manager.initBrowser();
     }
+
+    public static CloudAddressBook generateCloudAddressBook(AddressBook addressBook) {
+        List<CloudPerson> cloudPersonList = new ArrayList<>();
+        addressBook.getPersonList().stream()
+                .forEach( (p) -> cloudPersonList.add(new CloudPerson(p.getFirstName(), p.getLastName(), p.getId())));
```
Comment (damithc) [inline src/test/java/address/util/TestUtil.java]: Extra space before `(p)`, to be removed.

**Comment by damithc** [comment]
Looks good. 
May be you can add testing for Ctrl+Z too? I missed it out. I guess here we can test it for one or two cases just to ensure Ctrl+Z is bound to undo. More intensive testing of undo will be done in other gui tests.

**Comment by damithc** [comment]
@m133225 Did you added the Ctrl+Z testing but forgot to push. If not, it's ok I can add it.

**Comment by m133225** [comment]
Oops, I probably pushed it to my own repo instead.

**Pull request merged**